ecards/migrations/0001_initial.py

```diff
@@ -0,0 +1,63 @@
+                ('first_name', models.CharField(blank=True, max_length=150, verbose_name='first name')),
+                ('last_name', models.CharField(blank=True, max_length=150, verbose_name='last name')),
+                ('email', models.EmailField(blank=True, max_length=254, verbose_name='email address')),
+                ('is_staff', models.BooleanField(default=False, help_text='Designates whether the user can log into this admin site.', verbose_name='staff status')),
+                ('is_active', models.BooleanField(default=True, help_text='Designates whether this user should be treated as active. Unselect this instead of deleting accounts.', verbose_name='active')),
+                ('date_joined', models.DateTimeField(default=django.utils.timezone.now, verbose_name='date joined')),
+                ('following', models.ManyToManyField(related_name='followers', to=settings.AUTH_USER_MODEL)),
+                ('groups', models.ManyToManyField(blank=True, help_text='The groups this user belongs to. A user will get all permissions granted to each of their groups.', related_name='user_set', related_query_name='user', to='auth.group', verbose_name='groups')),
+                ('user_permissions', models.ManyToManyField(blank=True, help_text='Specific permissions for this user.', related_name='user_set', related_query_name='user', to='auth.permission', verbose_name='user permissions')),
+            ],
+            options={
+                'verbose_name': 'user',
+                'verbose_name_plural': 'users',
+                'abstract': False,
+            },
+            managers=[
+                ('objects', django.contrib.auth.models.UserManager()),
+            ],
+        ),
+        migrations.CreateModel(
+            name='Card',
+            fields=[
+                ('id', models.BigAutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
+                ('created_at', models.DateTimeField(auto_now_add=True)),
+                ('title', models.CharField(blank=True, max_length=255, null=True)),
+                ('message', models.CharField(max_length=255)),
+                ('font', models.TextField(choices=[("font-family: 'Cormorant Unicase', serif;", 'Cormorant Unicase'), ("font-family: 'Kaushan Script', cursive;", 'Kaushan Script'), ("font-family: 'Amatic SC', cursive;", 'Amatic SC'), ("font-family: 'Bungee Shade', cursive;", 'Bungee Shade'), ("font-family: 'Righteous', cursive;", 'Righteous'), ("font-family: 'Julius Sans One', sans-serif;", 'Julius Sans One'), ("font-family: 'Roboto', sans-serif;", 'Roboto'), ("font-family: 'Spectral', serif;", 'Spectral'), ("font-family: 'Dancing Script', cursive;", 'Dancing Script')], default="font-family: 'Roboto', sans-serif;")),
+                ('font_color', models.TextField(blank=True, null=True)),
```

Comment:
I think having this as null/blank might cause problems later, even if we do restrict it on the Front End. Can we go ahead and just define the default like we do for the other attributes?

Reply:
fixed